front/src/components/editable-cell/EditableRelation.tsx

```diff
@@ -95,7 +95,7 @@ function EditableRelation<
       }}
       editModeContent={
         <StyledEditModeContainer>
-          <StyledEditModeSelectedContainer>
+          <StyledEditModeSelectedContainer data-editmode="selected">
```

Comment:
I am not sure to understand why you are introducing this data-editmode attribute, it looks a bit hacky to me

Reply:
It's a way of ensuring that the 'x' doesn't show up on hover outside of the selected mode. Adding a data- prop seems like the least hacky way to get a known parent selector.